test/testmetrics/metrics-hpa/metric.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: metrics.keptn.sh/v1alpha3
+kind: KeptnMetricsProvider
+metadata:
+  name: my-provider
+  namespace: $NAMESPACE
```

Comment:
i think we can simplify the test by removing the `$NAMESPACE` place holder everywhere but from the `targetServer` URL